### ISSUE TITLE
Multi monitor docking

### ISSUE THREAD

**Issue by Mimerr** [Martenfur/TabletFriend#27]
Hi, I was just curious if there is a way to choose which monitor the docking uses when there are more than one?

I have a large monitor and separate from that a pen tablet. I am playing around with this as a way not to need my keyboard when using my tablet. But my large monitor is my main screen. So when I dock TabletFriend, it snaps to the large monitor instead of my tablet. I'd need it to dock to the tablet monitor.

Is the only way to do this to make the pen tablet the main monitor? Or is there a setting somewhere I am missing?

**Comment by Martenfur** [comment]
Nope, I literally never thought anyone would use this with multiple monitors.

**Comment by Mimerr** [comment]
Thanks for responding. No big deal. :D Most people will probably just use a keyboard when they're at their full setup so makes sense. I'm a sucker for customization and hotkey toolbars though. The more I use it, I kind of like it floating anyway so I'll probably just not worry about it.

**Comment by jcnegron** [comment]
I have the same issue. I'm on a desktop PC  and my tablet acts as a second monitor. I need the Toolbar docked in the second monitor which is the tablet, not to the main monitor. the work around is to leave it floating, in the second monitor but is sometimes obtrusive whenever a pop up window launches from the program.

**Comment by MorrisGunn** [comment]
Same here, I use my z13 as a drawing tablet and have an external monitor that I use for reference images and that sort of thing. the external monitor is main since I do a lot of gaming as well and switching it around each time i want to use Tablet Friend causes me to lose graphic settings.

**Comment by AlexFundorin** [comment]
That's a bummer. I have a touchscreen tablet and want to use the panel on it while having the main monitor as, well, the main monitor. Pressing "dock" button moves the app window to the main monitor. Can we have a fix or an option to chose the main monitor for the app?

**Comment by SeventhTower** [comment]
I know this post is a few years old, but I'm a digital artist and having this work on multiple monitors on my desktop sounded like a dream. So, I decided to fork it and take a crack at implementing it. You can now drag the toolbar when undocked to any screen/monitor and dock it accordingly to what ever screen it's on. Hopefully some of you might find some use to it 👍
I've only tested on Windows 11, as that is all that I have.

**Comment by Martenfur** [comment]
@SeventhTower Make a PR and I'll merge it, assuming it works.
